extensions/amp-fit-text/0.2/fit-text.css.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const LINE_HEIGHT_EM_ = 1.15;
+
+export const fitTextContent = {
+  'display': 'flex',
```

Comment:
Let's add `contain: layout paint` here as well.

Reply:
Made this `contain: content` since that's described as equivalent to `layout paint`: https://developer.mozilla.org/en-US/docs/Web/CSS/contain#Syntax